projects/pre-commit.com/package.yml

```diff
@@ -0,0 +1,47 @@
+distributable:
+  url: https://github.com/pre-commit/pre-commit/archive/refs/tags/v{{ version }}.tar.gz
+  strip-components: 1
+
+# if there’s a github then we can parse the versions
+versions:
+  github: pre-commit/pre-commit/releases/tags  # reads github *releases* but uses the tags of those releases
+  strip: /^v/
+
+dependencies:
+  python.org: '>=3.8'
+
+build:
+  dependencies:
+    pip.pypa.io: '*'
+    gnu.org/bash: '>=2'  # for our venv/relocation scripts: feel free to fix that
```

Comment:
actually no longer required for the generated script

Reply:
removed bash, pip still needed (could not build without it)